react-native-libraries.json

```diff
@@ -10763,5 +10763,12 @@
     "ios": true,
     "android": true,
     "expoGo": true
+  },
+  {
+    "githubUrl": "https://github.com/Acetyld/expo-foreground-actions",
+    "npmPkg": "expo-foreground-actions",
```

Comment:
Hello @Acetyld, thanks for adding new entry to the directory! 👍 

Just a nit before we ship, we can skip npm name in this case:
```suggestion
```